### ISSUE TITLE
Tests fail on RTX 3080

### ISSUE THREAD

**Issue by avive** [spacemeshos/gpu-post#51]
Linux binaries from release v0.1.20.

```
ubuntu@xxxx:~$ ./gpu-setup-test -t
Test LEAFS: Label size: 8, count 131072, buffer 0.1M
CPU: 131072 hashes, 5059 h/s
GeForce RTX 3080: 131072 hashes, 236592057 h/s
ZEROS result
WRONG result for label size 8 from provider 0 [GeForce RTX 3080]
ubuntu@xxxx:~$ ./gpu-setup-test -u
[GeForce RTX 3080]: hash test WRONG
ubuntu@xxxx:~$ nvidia-smi
Tue Jun 29 09:13:39 2021       
+-----------------------------------------------------------------------------+
| NVIDIA-SMI 460.80       Driver Version: 460.80       CUDA Version: 11.2     |
|-------------------------------+----------------------+----------------------+
| GPU  Name        Persistence-M| Bus-Id        Disp.A | Volatile Uncorr. ECC |
| Fan  Temp  Perf  Pwr:Usage/Cap|         Memory-Usage | GPU-Util  Compute M. |
|                               |                      |               MIG M. |
|===============================+======================+======================|
|   0  GeForce RTX 3080    Off  | 00000000:00:05.0 Off |                  N/A |
|  0%   29C    P0     1W / 320W |      0MiB / 10018MiB |      0%      Default |
|                               |                      |                  N/A |
+-------------------------------+----------------------+----------------------+
                                                                               
+-----------------------------------------------------------------------------+
| Processes:                                                                  |
|  GPU   GI   CI        PID   Type   Process name                  GPU Memory |
|        ID   ID                                                   Usage      |
|=============================================================================|
|  No running processes found                                                 |
+-----------------------------------------------------------------------------+
```

**Comment by AndrewAR2** [comment]
Fixed: added Nvidia Compute Capability 8.6 for support 3080.

**Comment by avive** [comment]
Fixed in release https://github.com/spacemeshos/gpu-post/releases/tag/v0.1.21
